Sources/Extensions/Foundation/URLExtensions.swift

```diff
@@ -8,6 +8,23 @@
 
 import Foundation
 
+// MARK: - Properties
+public extension URL {
+    
+    /// SwifterSwift: Dictionary of the URL's query parameters
+    public var queryParameters: [String: String]? {
+        guard let components = URLComponents(url: self, resolvingAgainstBaseURL: false), let queryItems = components.queryItems else { return nil }
+        
+        var items: [String: String] = [:]
+        
+        for queryItem in queryItems {
+            items[queryItem.name] = queryItem.value ?? ""
```

Comment:
any reason why we set an empty String? shouldn't it just be nil?

Reply:
cc @nathanbacon if you make this change I can get this merged in 👍

Reply:
@SD10, nil makes more sense. Changed this.